### ISSUE TITLE
Thank you boxes - add in related link

### ISSUE THREAD

**Issue by robynlloos** [uno-isqa-8950/uno-cpi#1453]
In addition to making thank you boxes consistent with modernized look, they should all provide links to repeat that step again. 

**For example:** After registering a new community partner (see screen shot for examples of both campus and community page) (same goes for campus partner page)
![image](https://user-images.githubusercontent.com/44984230/64448769-13634e80-d0a4-11e9-83e1-c27dad8d0fec.png)

- I should have the option to register another community partner on this page, because I may need to add more than 1 for a project I would like to enter.  *Suggest adding in linked text below the Thank You box that says “Register Another Community Partner” that takes them to this page: https://uno-cpi.herokuapp.com/partners/checkCommunityPartner/?
